omod/src/main/java/org/openmrs/module/locationbasedaccess/fragment/controller/manageAccount/ViewUserLocationFragmentController.java

```diff
@@ -0,0 +1,28 @@
+package org.openmrs.module.locationbasedaccess.fragment.controller.manageAccount;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.Location;
+import org.openmrs.User;
+import org.openmrs.api.context.Context;
+import org.openmrs.ui.framework.fragment.FragmentConfiguration;
+import org.openmrs.ui.framework.fragment.FragmentModel;
+
+public class ViewUserLocationFragmentController {
+
+    public void controller(FragmentModel model, FragmentConfiguration config) {
+        Object userId = config.getAttribute("userId");
```

Comment:
Did you intentionally decide to use the integer userId instead of user uuid?

Reply:
Just updated